src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Acquisition/MissingSetupComponentRegistrationService.cs

```diff
@@ -195,11 +226,22 @@ void RemoveConfiguration(Guid projectGuid, ConfiguredProject project)
                 {
                     _projectGuidToProjectConfigurationsMap.TryGetValue(projectGuid, out projectConfigurationSet);
                 }
-                
+
                 projectConfigurationSet?.Remove(project.ProjectConfiguration);
             }
         }
 
+        private void DisplayYellowBarInSolutionExplorer(ConfiguredProject project)
```

Comment:
Suggestion: `DisplayMissingComponentsPromptIfNeeded`.

This type doesn't know that the information will end up in a gold bar in the Solution Explorer; that's up to the `IMissingComponentRegistrationService`.

Reply:
done